clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/ClusterStateChangeTracker.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.clustercontroller.core;
+
+/**
+ * Class tracking whether we have changes in current and previous cluster stats.
+ *
+ * The state is considered changed if the current and previous cluster stats differs in whether
+ * they may have buckets pending in the 'global' bucket space. This signals that the ClusterStateBundle should be recomputed.
+ */
+public class ClusterStateChangeTracker {
```

Comment:
As per offline discussion, rename to `ClusterStatsChangeTracker` since `ClusterState` today is an established term for the actual cluster state instances.